### PR TITLE
Hnah108 refactor

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

```
☒ keyboards/handwired/hnah108/hnah108.h: LAYOUT_fullsize_iso: Nested layout macro detected. Matrix data not available!
```

@vuhopkep

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/hnah108/hnah108.c [hunk 1]
```diff
@@ -13,31 +13,33 @@
...
-  {   0,  51} ,                                                                                                                         { 205, 51},
-  {   0,  63} , { 18,  63}, { 37, 63},  { 56,  63}, { 74,  63},  { 93, 63}, { 112, 63}, { 130, 63},{ 149, 63}, { 168, 63}, { 186, 63 }, { 205, 63},
-}, {
-  1,1,1,1,1,1,1,1,1,1,1,1,1,
-  1,1,
-  1,1,
-  1,1,1,1,1,1,1,1,1,1,1,1,1,
-} };
\ No newline at end of file
+led_config_t g_led_config = {
```
Comment (drashna) [inline keyboards/handwired/hnah108/hnah108.c:19]:
```suggestion
#ifdef RGB_MATRIX_ENABLE
led_config_t g_led_config = {
```

File: keyboards/handwired/hnah108/hnah108.c [hunk 2]
```diff
@@ -13,31 +13,33 @@
...
+        {   0,  51 },                                                                                                                                             { 205,  51 },
+        {   0,  63 }, {  18,  63 }, {  37,  63 }, {  56,  63 }, {  74,  63 }, {  93,  63 }, { 112,  63 }, { 130,  63 }, { 149,  63 }, { 168,  63 }, { 186,  63 }, { 205,  63 }
+    }, {
+        // LED Index to Flag
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1,                                  1,
+        1,                                  1,
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1
+    }
+};
```
Comment (drashna) [inline keyboards/handwired/hnah108/hnah108.c:45]:
```suggestion
};
#endif
```

File: keyboards/handwired/hnah108/readme.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
 # hnah108
 
-![hnah108](https://i.imgur.com/rV0lM8M.jpg)  
+![hnah108](https://i.imgur.com/rV0lM8M.jpg)
```
Comment (drashna) [inline keyboards/handwired/hnah108/readme.md:3]:
```suggestion
![hnah108](https://i.imgur.com/rV0lM8Ml.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/hnah108/hnah108.c (hunk 1 above)
Comment (fauxpark) [reply keyboards/handwired/hnah108/hnah108.c:19]: Whoops

**Review by drashna — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**